chain/network/src/routing/edge_verifier_actor.rs

```diff
@@ -21,11 +21,17 @@ impl Handler<StopMsg> for EdgeVerifierActor {
     }
 }
 
-impl Handler<EdgeList> for EdgeVerifierActor {
+/// EdgeListToVerify contains list of Edges, and it's associated with a connected peer.
```

Comment:
nit: here you say 'ToVerify', but code says 'ToValidate'

Reply:
Fixed.